static/app/views/dashboardsV2/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -43,6 +45,18 @@ function WidgetCardContextMenu({
 
   const menuOptions: React.ReactNode[] = [];
 
+  if (isPreview) {
+    return (
+      <ContextWrapper>
+        <ContextMenu>
+          <PreviewMessage>
+            This is a preview only. To edit, you must add this dashboard
```

Comment:
nit: Add trailing period
```suggestion
            This is a preview only. To edit, you must add this dashboard.
```

Reply:
Also localize this message

```suggestion
            {t('This is a preview only. To edit, you must add this dashboard.')}
```